app/models/blueberry_cms/page.rb

```diff
@@ -34,10 +42,17 @@ def to_path
     def rebuild_path
       self.path_translations = I18n.available_locales.each_with_object({}) do |locale, translations|
         I18n.with_locale(locale) do
-          pages = self.ancestors_and_self.collect(&:slug).select(&:present?)
-          translations[locale] = '/' + pages.join('/')
+          translations[locale] = generate_path
         end
       end
     end
+
+    def generate_path
+      if parent
```

Comment:
Presne jak rikas... Mel jsem to puvodne jeste trochu jinak, pouzival jsem tam else, ale ted uz je  return vhodnejsi...